cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+## Cloth Diapers
+
+There are many sections that need to be written up in this document, but I'll
+start by writing up the critical sections required for basic knowledge
+or those that I have a specific opinion about.
+
+_Disclaimer_: These are my personal opinions and as such, don't come at me if
+yours are different!
+
+However, the plan is to include the following
++ Nomenclature (multiple languages)
++ [Types of Cloth Diapers](#types-of-cloth-diapers)
++ [Materials and Absorbency](#materials-and-absorbency)
++ [Newborn vs One Size](#newborn-vs-one-size)
++ Velcro vs Poppers
++ [Our Choices](#our-choices)
++ Things to Know
++ What causes leaks
+
+## Types of Cloth Diapers
+![image](https://user-images.githubusercontent.com/4185328/169092681-4b44aaef-1127-4937-b906-1b2ddc864821.png)
```

Comment:
This image has a massive copyright in the middle. "All rights reserved" ... 🙄 

![image](https://user-images.githubusercontent.com/4185328/169092681-4b44aaef-1127-4937-b906-1b2ddc864821.png)

Do you have permission to include it, or do you have a task in the backlog get our Graphic Designer to re-create it ASAP so that we don't fall foul of BS copyright law? 💭